.azure/regression-pipeline.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       test_case: 'kafka.*ST,mirrormaker.*ST'
       groups: 'regression'
       cluster_operator_install_type: 'bundle'
-      timeout: 360
```

Comment:
Are you sure this is working? I thought the hard timeout for each job is set to 6 hours by Azure itself.